packages/wonder-blocks-labeled-field/src/components/labeled-field.tsx

```diff
@@ -148,14 +174,29 @@ export default function LabeledField(props: Props) {
         return (
             <React.Fragment>
                 <Strut size={spacing.small_12} />
-                <LabelSmall
-                    style={light ? styles.lightError : styles.error}
-                    role="alert"
-                    id={errorId}
-                    testId={testId && `${testId}-error`}
-                >
-                    {error}
-                </LabelSmall>
+                <View style={styles.errorSection}>
+                    <PhosphorIcon
+                        icon={WarningCircle}
+                        style={[
+                            styles.errorIcon,
+                            light ? styles.lightError : styles.error,
+                        ]}
+                        role="img"
+                        aria-label={labels.errorIconAriaLabel}
+                    />
+                    <LabelSmall
+                        style={[
+                            styles.textWordBreak,
+                            styles.errorMessage,
+                            light ? styles.lightError : styles.error,
+                        ]}
+                        role="alert"
+                        id={errorId}
```

Comment:
The `aria-describedby` on the field uses this id for the error message. I'm going to update the PR so that it points to the whole error section so that the aria-label on the error icon is included!